githooks/pre-commit

```diff
@@ -2,53 +2,6 @@
 #
 # commit without checks by adding --no-verify to your commit command
 
-# run Black on all .py files
-printf "\n\e[1mRunning Black Python code formatter\e[0m\n"
-hash black 2>/dev/null || { echo >&2 "Black not installed in this environment. Halting commit."; exit 1; }
-black . --check
-black_exit=$?
-
-# run flake8 on all .py files
-printf "\n\e[1mRunning flake8 Python linter\e[0m\n"
-hash flake8 2>/dev/null || { echo >&2 "flake8 not installed in this environment. Halting commit."; exit 1; }
-flake8 . --count
-flake_exit=$?
-
-# run yamllint on all .yaml and .yml files
-printf "\n\e[1mRunning yamllint .yaml linter\e[0m\n"
-hash yamllint 2>/dev/null || { echo >&2 "yamllint not installed in this environment. Halting commit."; exit 1; }
-yamllint .
-yaml_exit=$?
-
-printf "\n\e[1mRunning git-crypt status and checking for warnings\e[0m\n"
-# git-crypt check for unencrypted file warnings adapted from https://www.gitmemory.com/issue/AGWA/git-crypt/45/614607121
-if [ -d ".git-crypt" ]; then
-    hash git-crypt status | grep "\*\*\* WARNING"; test $? -eq 1
-fi
-printf "git-crypt fine\n\n"
-
-exec < /dev/tty
-
-if [ $black_exit -gt 0 ]; then 
-    printf "\e[1mBLACK WARNING\e[0m: not all of your Python files comply with Black\n"
-    printf "You might want to run Black on the files listed above before committing\n"
-fi
-
-if [ $flake_exit -gt 0 ]; then 
-    printf "\e[1mFLAKE8 WARNING\e[0m: not all of your Python files comply with Flake8\n"
-    printf "Please check if you want to make changes before committing\n"
-fi
-
-if [ $yaml_exit -gt 0 ]; then 
-    printf "\e[1mYAML WARNING\e[0m: not all of your .yaml files comply with yamllint\n"
-    printf "Please check if you want to make changes before committing\n"
-fi
-
-echo ""
-while read -p "Continue with commit? (Y/n) " yn; do
-    case $yn in
-        [Yy] ) printf "Continuing with commit\n"; break;;
-        [Nn] ) printf "Cancelling commit\n"; exit 1;;
-        * ) echo "Please answer y (yes) or n (no):" && continue;
-    esac
-done
+# add .version
+poetry version | awk '{print "__version__ = \"" $2 "\""}' > arrow_pd_parser/__init__.py
+git add arrow_pd_parser/__init__.py
```

Comment:
don't want to keep the linting and that? 🧐

Reply:
Nope